cli/command_restore.go

```diff
@@ -96,6 +98,34 @@ type restoreSourceTarget struct {
 	isplaceholder bool
 }
 
+type objectReaderOptionsFlags struct {
+	disableReadAhead bool
+	readAheadMB      int64
```

Comment:
What is the behavior of the system when `readAheadMB == 0` and disableReadAhead == false`? Is it necessary to have the flag variable?